config/manager/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+resources:
+- manager.yaml
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+configMapGenerator:
+- files:
+  - controller_manager_config.yaml
+  name: manager-config
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: ajazee/harborconfiguration
```

Comment:
Where is this image coming from? I guess built manually?

Reply:
@MarcelMue A question towards this: should this app be build and pushed with architect-orb? If so, is this expected in this or coming PRs?